javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+      this.currentTime = 0;
+      this.intervalId = 0;
+   }
+
   startClick(callback) {
-    // ... your code goes here
+    let intervalID = setInterval(() => {
+      this.currentTime++;
+    }, 1000)
   }
   getMinutes() {
-    // ... your code goes here
+    let minutes = this.currentTime/60
+    minutes = Math.floor(minutes)
+    return minutes
   }
   getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
+    let seconds = this.currentTime % 60
+    seconds = Math.floor(seconds)
+    return seconds
+  }
+
+  twoDigitsNumber(number) {
+    if (number.toString().length < 2) {
+      number = "0" + number;
+    }
+    return number
+  }
+
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.intervalID);
   }
   resetClick() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
   splitClick() {
-    // ... your code goes here
+    return `${this.twoDigitsNumber(this.getMinutes())}:${this.twoDigitsNumber(this.getSeconds())}`
```

Comment:
this works perfect, but to have a clearer code, you may want to save the function calls inside variables, and then return the interpolation of the variables. It is more lines of code but not always less lines is better.